app/src/main/java/info/nightscout/androidaps/MainActivity.kt

```diff
@@ -266,9 +274,16 @@ class MainActivity : NoSplashAppCompatActivity() {
         return super.dispatchTouchEvent(event)
     }
 
+    private fun setPluginPreferenceMenuName() {
+        val plugin = (main_pager.adapter as TabPageAdapter).getPluginAt(main_pager.currentItem)
+        this.menu?.findItem(R.id.nav_plugin_preferences)?.title = plugin.name + ' ' + resourceHelper.gs(R.string.nav_preferences)
```

Comment:
this is not possible translate correctly
Use %s

Reply:
Can you explain what you mean with %s ? The function here replaces the title with concetinating 3 Strings.
And ist worked well. 
I dont know what is Not possible ?